cephmetrics/build/build_rpm

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/bash
+set -ex
+
+# Only do actual work when we are an RPM distro
+if test "$DISTRO" != "fedora" -a "$DISTRO" != "centos" -a "$DISTRO" != "rhel"; then
+    exit 0
+fi
+
+
+## Install any setup-time deps (to make dist package)
+
+# We need this to get the major version from lsb_release
+yum install -y redhat-lsb-core mock git
+
+# Run the install-deps.sh upstream script if it exists
+if [ -x install-deps.sh ]; then
+    echo "Ensuring dependencies are installed"
+    sudo ./install-deps.sh
+fi
+
+
+## Get some basic information about the system and the repository
+RELEASE="$(lsb_release --short -r | cut -d '.' -f 1)" # sytem release
+VERSION="0.1"
+REVISION="$(git rev-list --count HEAD)-g$(git rev-parse --short HEAD)"
+RPM_RELEASE=$(echo $REVISION | tr '-' '_') # the '-' has a special meaning
+
+
+## Build the source tarball
+echo "Building source distribution"
+git archive --format=zip --prefix=cephmetrics-${VERSION}/ HEAD > dist/cephmetrics-${VERSION}.zip
+wget https://grafana.com/api/plugins/vonage-status-panel/versions/1.0.4/download -O dist/vonage-status-panel-1.0.4.zip
```

Comment:
It is another source, it is for a grafana plugin called vonage-status-panel that cephmetrics requires. I did not want to split this into two packages so I just renamed the module to a unique name (cephmetrics-vonage) to avoid conflicts and packaged it in a separate subpackage.